import psycopg2 as pg

try:
    con = pg.connect(
        database="projeto",
        user="postgres",
        password="postgres",
        host="localhost",
        port="5432"
    )
    print("conex√£o realizada")
    con.close()
except Exception as erro:
    print(erro)